polygeist/include/mlir/Conversion/PolygeistPasses.td

```diff
@@ -0,0 +1,53 @@
+#ifndef MLIR_CONVERSION_POLYGEISTPASSES
+#define MLIR_CONVERSION_POLYGEISTPASSES
+
+include "mlir/Pass/PassBase.td"
+
+def ConvertPolygeistToLLVM : Pass<"convert-polygeist-to-llvm", "mlir::ModuleOp"> {
+  let summary = "Convert scalar and vector operations from the Standard to the "
```

Comment:
Is the comment accurate ?  I would expect a pass named `ConvertPolygeistToLLVM` to convert polygeist operations to LLVM. Other dialect operations should be converted by their respective conversion passes.

Reply:
Fixed doc.